home/dfhack/www/builds/index.html

```diff
@@ -11,16 +11,24 @@
         <h1>DFHack builds</h1>
         <section>
             <h2>Stable and pre-release builds</h2>
-            <p>Stable DFHack builds and pre-releases are available on <a href="https://github.com/dfhack/dfhack/releases">GitHub</a>.</p>
+            <p>Stable DFHack builds and pre-releases are available on <a href="https://store.steampowered.com/app/2346660/DFHack">Steam</a> and <a href="https://github.com/dfhack/dfhack/releases">GitHub</a>.</p>
             <p><a href="https://github.com/dfhack/dfhack/releases" class="big-ol-button">Download Latest DFHack&nbsp;Release</a></p>
         </section>
         <section>
             <h2>Development builds (potentially unstable)</h2>
             <p>
                 <span style="color: #e00">Please only download these builds if you plan on testing them and providing feedback.</span> <a href="https://docs.dfhack.org/en/latest/docs/Introduction.html#support">See our documentation</a> for ways to provide feedback and/or get involved.
             </p>
-            <p>For most users, we recommend the pre-release builds on <a href="https://github.com/dfhack/dfhack/releases">GitHub</a> instead.</p>
-            <p><a href="https://files.dfhack.org/nightly/">Download nightly builds</a></p>
+            <p>For most users, we recommend the pre-release builds on <a href="https://store.steampowered.com/app/2346660/DFHack">Steam</a> instead.</p>
+            <p>
+                Instructions for downloading bleeding-edge development builds:
```

Comment:
We should have these instructions in our docs as well. Maybe this could be a link to there?

Reply:
That's a good idea. I copied these from an inferior documentation source: https://discord.com/channels/793331351645323264/807444616731295755/1067898346406162474